www/source/blog/2018-02-21-Habitat-Helm.html.md

```diff
@@ -57,24 +44,24 @@ Once you've cloned core-plans locally, go to `core-plans/nginx` and enter the Ha
 
 * Enter the Habitat Studio within the nginx core plan with `hab studio enter`
 * Build the latest nginx package and stores it as a habitat artifact (“hart”) in the nginx plan’s `results` directory: `build`
-* Export the hab artifact to a Helm chart: `hab pkg export helm results/<yourfilename>.hart` This command will look very similar to `hab pkg export helm results/tdrew-nginx-1.11.10-20180221053714-x86_64-linux.hart`
+* Initialize the Helm client-side: `helm init -c`
```

Comment:
Is this really needed?

Reply:
As I explained in the PR description, with the current layout, yes. The question is if we want to make big changes (moving sections and add new sections) to this post?

Reply:
Feel free to ask Tasha on the upstream PR?

Reply:
Sure.

Reply:
I am a bit confused by this, can you even do helm while inside the studio?

Reply:
I'm not sure but I will try once I can get the studio working here.